### PR TITLE
ci(.github/workflows): removes pre-release workflow.

### PR DESCRIPTION
**Pull request by matthewhartstonge**
TSIA!

### CODE REVIEW

File: .github/workflows/pre-release.yml [hunk 1]
```diff
@@ -18,16 +18,12 @@ jobs:
...
         id: release-please
         with:
-          package-name: "@linc-technologies/ember-paper-password"
-          release-type: "node"
-          bootstrap-sha: "80dd68e536489b80855b0b638d675874f0f0b009"
-          versioning-strategy: "prerelease"
-          prerelease: true
-          skip-github-release: false
-          default-branch: "beta"
+          manifest-file: "./release-please-config.prerelease.json"
```
Comment (matthewhartstonge) [inline .github/workflows/pre-release.yml:24]: really wanna move all configy files into `.github`

Comment (hero-david) [reply .github/workflows/pre-release.yml:24]: Why not just have it as the default manifest file name and update the pre-release tag later?

Comment (matthewhartstonge) [reply .github/workflows/pre-release.yml:24]: Everything has to be configured via a manifest file now :/

Comment (hero-david) [reply .github/workflows/pre-release.yml:24]: Sure, but isnt the default manifest name `.release-please-manifest.json`, could leave the option out if we keep with the expected filename:
```
      # optional. customize path to .release-please-manifest.json
      manifest-file: .release-please-manifest.json
```

It'll be easier to diff the file if the name doesnt change and we just edit the contents (When setting prerelease false), else it'll show as a removal and addition of a file

**Review by hero-david — APPROVED** [review]
We nattered and this is probably overkill, if we do want to have prerelease projects for consumption via NPM, we can revisit this.

In the meantime, for out of band testing, we can pull via Git ref on an internal beta branch

**Pull request merged**